.github/workflows/lint.yml

```diff
@@ -0,0 +1,20 @@
+name: Lint
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  lint:
+    runs-on: ubuntu-20.04
+    name: lint
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Lint
+        run: make lint
```

Comment:
sorry for the dumb question.. but does `make lint` rewrite the file?

Reply:
I dont think so, why ?

Reply:
Ignore the question.. Some linters support `autofix` but in this case we don't commit the newer files anyway.
https://golangci-lint.run/usage/linters/

Reply:
Ahhh OK, yeah we dont support that..at least not yet. Usually I prefer for people to detect this locally (like pre-commit hook) and then just apply themselves (we could have make automation for that). But would not use on CI.